### ISSUE TITLE
Add island and islandGroup to fields that can be pushed to GEOLocate CoGe

### ISSUE THREAD

**Issue by themerekat** [Symbiota/Symbiota#1928]

**Comment by themerekat** [comment]
This is already done, but it seems the fields still aren't showing up in CoGe

**Comment by NikitaSalikov** [comment]
Hmm, I thought Nelson saw those fields on their end, judging by the email conversation.

**Comment by themerekat** [comment]
> Hmm, I thought Nelson saw those fields on their end, judging by the email conversation.

Right, I think they're going in the database, but I'm just not seeing them in their UI. So, I think for our purposes, this is closed.
